src/core/core.scale.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Why would you disable eslint for this file, you should resolve the error instead